restaurant-front/src/modules/chef/pages/chef-dashboard/chef-dashboard.component.ts

```diff
@@ -0,0 +1,250 @@
+import { BreakpointObserver } from '@angular/cdk/layout';
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatSidenav } from '@angular/material/sidenav';
+import { Router } from '@angular/router';
+import { ToastrService } from 'ngx-toastr';
+import { BehaviorSubject } from 'rxjs';
+import { UserList } from 'src/modules/shared/models/user-list';
+import { UserWithBadgeNum } from 'src/modules/shared/models/user-with-badgenum';
+import { UserWithToken } from 'src/modules/shared/models/user-with-token';
+import { NotificationService } from 'src/modules/shared/services/notification-service/notification.service';
+import { UserService } from 'src/modules/shared/services/user-service/user.service';
+
+@Component({
+  selector: 'app-chef-dashboard',
+  templateUrl: './chef-dashboard.component.html',
+  styleUrls: ['./chef-dashboard.component.scss']
+})
+export class ChefDashboardComponent implements OnInit {
+
+  @ViewChild(MatSidenav)
+  sidenav!: MatSidenav;
+  pageSize: number;
+  currentPage: number;
+  data: any[];
+  imagePath: string;
+  user: UserWithToken;
+  showModalPasswordChange: boolean;
+  showModalLogout: boolean;
+  waiterList: Array<UserWithBadgeNum>;
+  showModalOtherAccounts: boolean;
+  showModalLogin: boolean;
+  currentBadgeContent: number;
+
+  
+  isDishView:boolean = false;
+  isNewOrders: boolean =false;
+  isAcceptedOrders: boolean =false;
+  isNewOrderItems: boolean =false;
+  isAcceptedOrderItems: boolean =false;
+  idOrder: any;
```

Comment:
Same as with the other dashboard, try having a single state string or even two that you will change in (click) event.